### PR TITLE
Update third-party-join.md

### PR DESCRIPTION
**Pull request by NancyForero**
adding the new supported partner in a paragraph that was not merged before

### CODE REVIEW

File: Teams/rooms/third-party-join.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ description: This article discusses how to configure your organization and Teams
 
 # Enable Teams Rooms devices to join third-party meetings
 
-Microsoft Teams Rooms devices support a one-touch experience for joining third-party online meetings, also referred to as Direct Guest Join. When enabled, you can use Teams Rooms to join meetings hosted on Cisco Webex and Zoom just as easily as you can join meetings hosted in Microsoft Teams. Keep in mind that in this case, the experience in Teams Rooms of the third-party online meeting is driven by the third-party online meeting provider.
+Microsoft Teams Rooms devices support a one-touch experience for joining third-party online meetings, also referred to as Direct Guest Join. When enabled, you can use Teams Rooms to join meetings hosted on BlueJeans by Verizon, Cisco Webex or Zoom just as easily as you can join meetings hosted in Microsoft Teams. Keep in mind that in this case, the experience in Teams Rooms of the third-party online meeting is driven by the third-party online meeting provider.
```
Comment (JohanFreelancer9) [inline Teams/rooms/third-party-join.md:24]:
```suggestion
Microsoft Teams Rooms devices support a one-touch experience for joining third-party online meetings, also referred to as Direct Guest Join. When enabled, you can use Teams Rooms to join meetings hosted on BlueJeans by Verizon, Cisco Webex, or Zoom just as easily as you can join meetings hosted in Microsoft Teams. Keep in mind that in this case, the experience in Teams Rooms of the third-party online meeting is driven by the third-party online meeting provider.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NancyForero, kindly commit the minor suggested change. Many thanks.

**Comment by scanum** [comment]
@tonysmit 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by NancyForero — COMMENTED** [review]
please reject changes. BlueJeans will close business soon and we will stop supporting it

**Pull request closed without merging**